CHANGELOG-1.1.md

```diff
@@ -1,3 +1,61 @@
+# TiDB Operator v1.1.0-rc.1 Release Notes
+
+This is a release candidate of `v1.1.0`, which focuses on the usability, extensibility and security of TiDB Operator. While we encourage usage in non-critical environments, it is **NOT** recommended to use this version in critical environments.
+
+## Action Required
+
+- Add field `tlsClient` in the backup and restore spec to control whether to enable TLS for connection with TiDB Server and also support specifying a secret name that including the cert. ([#2003](https://github.com/pingcap/tidb-operator/pull/2003), [@shuijing198799](https://github.com/shuijing198799))
+- change the field `BRConfig.EnableTLSClient` as type `bool` to `BRConfig.TLSCluster` as type `*TLSCluster` as the TLS trigger. ([#1988](https://github.com/pingcap/tidb-operator/pull/1988), [@shuijing198799](https://github.com/shuijing198799))
+
+
+## Other Notable Changes
+
+- Fix the PD `location-labels` configuration ([#1941](https://github.com/pingcap/tidb-operator/pull/1941), [@aylei](https://github.com/aylei))
+- Able to pause and unpause tidb cluster deployment via `spec.paused`. ([#2013](https://github.com/pingcap/tidb-operator/pull/2013), [@cofyc](https://github.com/cofyc))
+- Default the `max-size` and `max-backups` for tidb server configuration with `300` and `3` if the tidb cluster was deployed by CR. ([#2045](https://github.com/pingcap/tidb-operator/pull/2045), [@Yisaer](https://github.com/Yisaer))
+- Able to configure custom env for components. ([#2052](https://github.com/pingcap/tidb-operator/pull/2052), [@cofyc](https://github.com/cofyc))
+- Fix the error that `kubectl get tc` cannot show correct images ([#2031](https://github.com/pingcap/tidb-operator/pull/2031), [@Yisaer](https://github.com/Yisaer))
+- 1. Default the `spec.tikv.maxfailureOver` and `spec.tidb.maxfailureOver` to 3 when they are no defined
+  2. Change the action that when `maxfailureOver` is zero, operator would not failover instead of creating limitless instances. ([#2015](https://github.com/pingcap/tidb-operator/pull/2015), [@Yisaer](https://github.com/Yisaer))
```

Comment:
@Yisaer This sentence is confusing. Could you pls explain?

Reply:
I'm going to simplify it to "Disable auto-failover when `maxFailoverCount` is set to `0`".